### ISSUE TITLE
Apple tv support.

### ISSUE THREAD

**Issue by anjandaffo** [cocos2d/cocos2d-x#15418]
I downloaded the cocos2d-x version 3.10 and it's not supporting the apple tv.

Previously I created a app using cocos2d-x and now I need to support same app for apple tv. currently it supporting android and iOS.

Please suggest me how to cocos2d-x app run on Apple Tv.

Are you planning to give the Apple Tv support on next version i.e 3.11.

Cocos2d-x test project is also not supporting Apple Tv.

is cocos2d-x app is running on Apple Tv simulator?

Kindly consider.

**Comment by slackmoehrle** [comment]
Please use our forums to research: http://discuss.cocos2d-x.org/

Here are two threads where users are doing it successfully.

http://discuss.cocos2d-x.org/t/building-for-the-new-appletv-tvos/23758

http://discuss.cocos2d-x.org/t/appletv-support/25416

**Comment by minggo** [comment]
Can close this issue?

**Comment by anjandaffo** [comment]
@minggo I gone through the above link and it working. But there is an issue to update cocos2d-x. In above repos use cocos2d-x 3.9. If I want to update the cocos2d to next version then build is failed because library are not building for Apple TV.

So I request If Cocos2d team give official support It will help to many users.

**Comment by minggo** [comment]
@anjandaffo I know nothing about Apple TV support. I think @ricardoquesada knows more about it.

**Comment by ricardoquesada** [comment]
yes, I started working on AppleTV support, but then priorities shifted to other things.
I was able to run the test-cpp on the tvOS simulator, but I didn't test them on the real device since I don't have one.

**Comment by anjandaffo** [comment]
@ricardoquesada  So Can We get Apple tv support in cocos2d-x next version 3.11? How you test menuItem or button click on simulator?

**Comment by ricardoquesada** [comment]
@anjandaffo It won't be part of v3.11.
The only thing that I tried was running cpp-test, and just that. Most probably the buttons don't work since we haven't added remote control support.
